packages/react-router/index.ts

```diff
@@ -289,7 +356,7 @@ function mapRouteProperties(route: RouteObject) {
 }
 
 export function createMemoryRouter(
-  routes: RouteObject[],
+  routes: AgnosticRouteObject[],
```

Comment:
Did TS not like this?  We want react-aware objects at this point when we create a router since they need elements on them so I think we want to revert this change